src/apps/explorer/pages/TransactionDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useParams } from 'react-router'
+
+import { BlockExplorerLink } from 'components/common/BlockExplorerLink'
+import { TitleAddress, WrapperPage } from 'apps/explorer/pages/styled'
+import { useNetworkId } from 'state/network'
+import { TransactionsTableWidget } from 'apps/explorer/components/TransactionsTableWidget'
+
+const TransactionDetails: React.FC = () => {
```

Comment:
I have taken the structure from the [PR882](https://github.com/gnosis/gp-ui/pull/882) to synchronize us @maria-vslvn